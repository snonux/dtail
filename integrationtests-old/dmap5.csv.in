count($time),$time,max($goroutines),avg($goroutines),min($goroutines)
23,1002-071147,16.000000,14.391304,12.000000
20,1002-071213,17.000000,14.100000,12.000000
20,1002-071143,17.000000,15.000000,13.000000
