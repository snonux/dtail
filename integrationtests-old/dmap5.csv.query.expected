select sum($timecount),last($time),min($min_goroutines), group by $hostname set $timecount = `count($time)`, $time = `$time`, $min_goroutines = `min($goroutines)` logformat csv outfile dmap5.csv.tmp